From 32b0a517965fcc166583fb288e46be7b1ffc4925 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Wed, 11 Oct 2017 15:50:07 -0500
Subject: [PATCH 2638/5242] MLK-16571-1: Kconfig: arm64: enable lpspi for
 MXC_ARM64

commit  06e4f17acdeeb188d31714fae173e603a9e7a8fd from
https://source.codeaurora.org/external/imx/linux-imx.git

enable the lpspi config for arm64 in Kconfig

BuildInfo: SCFW 9e9f6ec6, IMX-MKIMAGE 0, ATF 0

Reviewed-by: Pan Gao <pandy.gao@nxp.com>
Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/spi/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/Kconfig b/drivers/spi/Kconfig
index ad5d68e..2f1315e 100644
--- a/drivers/spi/Kconfig
+++ b/drivers/spi/Kconfig
@@ -247,7 +247,7 @@ config SPI_FALCON
 
 config SPI_FSL_LPSPI
 	tristate "Freescale i.MX LPSPI controller"
-	depends on ARCH_MXC || COMPILE_TEST
+	depends on ARCH_MXC || ARCH_MXC_ARM64 || COMPILE_TEST
 	help
 	  This enables Freescale i.MX LPSPI controllers in master mode.
 
-- 
1.7.9.5

